modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -946,6 +945,24 @@ private boolean _isReverse(
 		return true;
 	}
 
+	private ObjectEntry _parseToDTOObjectEntry(
```

Comment:
Maybe the method should be renamed to _toObjectEntry, and also the parameters could be renamed to just properties? It does not make sense to name like that if this method should not be aware of if it is a nestedObjectEntry or not